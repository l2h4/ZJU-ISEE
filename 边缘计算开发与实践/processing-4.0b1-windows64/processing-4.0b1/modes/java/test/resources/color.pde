boolean test = true;
color c1 = color(255, 255, 255);
color c2 = test ? #A011CD : #C0C0C0C0;