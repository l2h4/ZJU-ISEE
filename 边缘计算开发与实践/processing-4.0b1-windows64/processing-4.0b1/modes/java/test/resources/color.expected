import processing.core.*;
import processing.data.*;
import processing.event.*;
import processing.opengl.*;

import java.util.HashMap;
import java.util.ArrayList;
import java.io.File;
import java.io.BufferedReader;
import java.io.PrintWriter;
import java.io.InputStream;
import java.io.OutputStream;
import java.io.IOException;

public class color extends PApplet {

    public void setup() {
boolean test = true;
int c1 = color(255, 255, 255);
int c2 = test ? 0xFFA011CD : 0xC0C0C0C0;
    noLoop();
    }

    static public void main(String[] passedArgs) {
        String[] appletArgs = new String[] { "color" };
        if (passedArgs != null) {
            PApplet.main(concat(appletArgs, passedArgs));
        } else {
            PApplet.main(appletArgs);
        }
    }
}
